runtime/onert/backend/train/ops/OperationUtils.cc

```diff
@@ -16,9 +16,184 @@
 
 #include "OperationUtils.h"
 
+#include <cker/eigen/EigenSupport.h>
+#include <cker/eigen/Utils.h>
+#include <cker/operation/Helper/Tensor.h>
 #include <cker/train/operation/ReLU.h>
 #include <cker/train/operation/ReLU6.h>
 
+// From tensorflow/core/kernels/redux_functor.h
```

Comment:
Please create a separate file for this change.

Reply:
I've created a separate file named `ReduxFunctor.h`.